### PR TITLE
12-15-2020 equity dashboard chart data update

### PR DESCRIPTION
**Pull request by wpserviceuser**
Equity dashboard stats updates in this PR may be reviewed on staging: https://staging.covid19.ca.gov/equity/

After reviewing, if all looks well, approve and merge this Pull Request.

If there are issues with the data:

- Note concerns or issues here by commenting on this PR

- Work with Triston directly to resolve data issues

- Alert the COVID19 site team in Slack (in the Equity page channel)

### CODE REVIEW

**Comment by vargoCDPH** [comment]
- statewide case rate in top boxes did not change from yesterday???
- similarly, statewide case rate on the social determinants charts is identical to yesterday, not unreasonable
@sindhuravuri  I'll wait for your review.

**Comment by sindhuravuri** [comment]
Hi everyone! - Here are notes from data review for today, @vargoCDPH please do modify/adjust as you see fit!

(Before I start, about layout - love the new navigation “on this page” at the top! Very helpful)

1. Blue Box At the Top Updates

1a. All four blue box updates (for cases/100K) align with calculated percentages in orange
1b. In contrast to previous 24h (12/13 → 12/14), NHPI and Median Income cases/100K (white text in boxes) decreased noticeably in last 24h, while “all ethnicities”/”all income brackets” case number/100K remained stable (exact same number as previous 24h)
1c. Higher rate of increase seen in "Death rate for black people" (62.3 → 62.7), compared to that seen between 12/13 & 12/14 (62.2 → 62.3). 
1d. “Case rate for Latino people” oscillating between 8 and 9% over past 48h

2. Statewide death rate/100K remains stable in past 48h at 5.4 — NHPI stable in past 24h (in contrast to noticeable decrease from previous 24h), but White and Black return to exact same values as 12/13 (past 24h shows exact opposite trend in Black/White statewide death rate/100K than previous 24h, while NHPI remains stable; see values below for specifics)

2a. NHPI (12/13  → 12/14 → 12/15): 9.6 → 8.8 → 8.8 
      Black (12/13  → 12/14 → 12/15): 5.6 → 5.4 → 5.6 
      White  (12/13  → 12/14 → 12/15): 5.3 → 5.4 → 5.3
2b. Deaths relative to percentage of population in CA data: White % of COVID deaths statewide (yellow bar) **decreased** by .8% since yesterday, while all other race/ethnicity’s % of statewide COVID deaths **increased** (by ≤.4%) or stayed stable/did not change significantly

3. Statewide case rate/100K increased noticeably in past 24h, at similar pace as 12/13 to 12/14 jump — NHPI increased at much faster pace between 12/14 → 12/15 than between 12/13 → 12/14

3a. Cases relative to percentage of population in CA: did not change significantly (NHPI stayed stable) - **Only potential discrepancy - White % cases statewide (yellow bar) increased by .2%, while death rate decreased by .8% (see point 2d)**
3b. Unknown category went from “9.2% increase since previous month” on 12/12 to “10.0% increase since previous month” on 12/14 to now “10.9% increase since previous month” (continues in large increases)

4. In contrast to previous 72h, testing rate/100K increased at faster pace for race/ethnicities (i.e., increases in NHPI, White, Black, and Multi-Race = lowest rate), while statewide tests/100K increased at rate similar to previous 72h

5. Mortality rate per 100K < Case rate per 100K (as expected)

6. Community case rate/100K by median annual household income bracket increased noticeably in past 24h: increases in 0-40 (faster pace as previous 24h) and 40-60 (same pace than previous 24h); 0-40 remained highest, as expected

**6a. Statewide case rate on social determinants chart remains the same despite visible increases in statewide cases/100K — not expected**

7. HE metric: did not extend from yesterday, last shown date still Nov. 29 — increase in health equity quartile positivity slowed to .1% in past 24h (in contrast to .5% in previous 48h), while statewide positivity did not change at all (similar to relatively stable trends seen in past 72h) 

8. Data completeness: in contrast to noticeable decrease seen in previous 24h, all three categories (R&E, Sexual orientation, Gender identity) remain largely stable in data completeness for death; in alignment with previous 24h, all three categories show decreases in data completeness for cases (largest/noticeable decrease is 0.8% seen in R&E, followed by 0.5% for SO and 0.1% for GI)

**Comment by vargoCDPH** [comment]
noticed that the statewide case rate on the bottom social determinant graphs did not change from yesterday but nearly every other state wide rate number on the page did go up. Is that correct?  do you have an explanation for what could be going on there?

**Comment by abquirarte** [comment]
We've been referring to the salmon instance for more accurate info. @James
Bumgardner <james.bumgardner@digital.ca.gov> can you dig in? Let me know if
we need to ping Aaron?

**Comment by vargoCDPH** [comment]
@abquirarte does this mean Salmon and staging could be different?

**Comment by abquirarte** [comment]
That's my suspicion! (Which we'll need to dig and verify)

File: data/reviewed/equitydash/equitytopboxdatav2.json [hunk 1]
```diff
@@ -1,77 +1,77 @@
 {
   "LowIncome": [
     {
-      "DATE": "2020-12-14",
+      "DATE": "2020-12-15",
       "STATE_CASE_RATE_PER_100K": 3869.37080168137,
```
Comment (coreycaitlin) [inline data/reviewed/equitydash/equitytopboxdatav2.json:5]: Where (which file/query/snowflake task) do we get `STATE_CASE_RATE_PER_100K`? This number should have changed.

**Comment by vargoCDPH** [comment]
On Salmon the top blue box rates did change from the live site, but the Health Equity metric lines appear the same to the live site. And on staging the top blue boxes did not change

**Comment by abquirarte** [comment]
Uh oh! Well, we have a sync issue here. 🤨. Need to fix/confirm this before
Thurs announcement.
@Corey Mahoney <corey.mahoney@digital.ca.gov> @James Bumgardner
<james.bumgardner@digital.ca.gov>

**Comment by coreycaitlin** [comment]
Staging and Salmon should not be different. Blue top boxes only _half_ updated — doing full code-level review now so we can trace back to SQL queries and ❄️  tasks.

**Comment by vargoCDPH** [comment]
from Triston:
`Looks as though the cumall file (file produced daily by IRA) wasn't updated by the script run time (which is where the statewide rate is pulled from) while the testing_case_tract_calc.tsv file had.  I dug through older copies of the cumall file (they keep .RDS versions for past dates) and compared them against the tables I've sent over the past two weeks, and this is the first time it's occurred.  I've added date checks in the script tonight so that the tables won't be auto-created if the state date isn't congruent with the rest of the table.`

File: data/reviewed/equitydash/social-data-income.json [hunk 2]
```diff
@@ -1,74 +1,74 @@
...
+    "DATE": "2020-12-15",
     "SOCIAL_DET": "income",
     "SOCIAL_TIER": "above $120K",
     "SORT": 0,
-    "CASES_7DAYAVG_7DAYSAGO": 629.857142857143,
+    "CASES_7DAYAVG_7DAYSAGO": 647.428571428571,
     "POPULATION": 2744855,
-    "CASE_RATE_PER_100K": 22.9468275321335,
+    "CASE_RATE_PER_100K": 23.5869862498592,
     "STATE_CASE_RATE_PER_100K": 63.899739170789,
```
Comment (coreycaitlin) [inline data/reviewed/equitydash/social-data-income.json:10]: This also should have changed.

**Review by coreycaitlin — COMMENTED** [review]
Statewide/all-ethnicities numbers did not update as expected in several spots. Tracing back through the flow to track down why.

**Comment by coreycaitlin** [comment]
Screen Shot 2020-12-15 at 9 39 33 PM

- Salmon environment isn't getting updated by either "in-review" data or "reviewed" data, so it's oldest (numbers are smallest)
- Prod (in the middle) has "reviewed" data from yesterday
- Staging environment is getting updated by "in-review" data; the demographic numbers changed, but (as @sindhuravuri spotted) the statewide numbers *didn't change* where they should have

File: data/reviewed/equitydash/social-data-income.json (hunk 2 above)
Comment (jbum) [reply data/reviewed/equitydash/social-data-income.json:10]: Let me know (via Slack) if you want me to rerun the Cron.

Comment (jbum) [reply data/reviewed/equitydash/social-data-income.json:10]: Salmon / Lemon pond are likely out of sync today.

File: data/reviewed/equitydash/equitytopboxdatav2.json (hunk 1 above)
Comment (jbum) [reply data/reviewed/equitydash/equitytopboxdatav2.json:5]: There are two queries that produce a number with that label.  casesLowIncome, and socialData. The first pulls from COVID.PRODUCTION.VW_CDPH_CASE_RATE_BY_SOCIAL_DET and the second from 
PRODUCTION.VW_CDPH_CASE_RATE_BY_SOCIAL_DET

Comment (jbum) [reply data/reviewed/equitydash/equitytopboxdatav2.json:5]: If you have snowflake access, I can send you the actual queries - you can run them in the Worksheet tab.

**Comment by coreycaitlin** [comment]
Closing this out. We're gonna try again tomorrow, and review the updates closely.

**Comment by abquirarte** [comment]
Thank you folks!

**Pull request closed without merging**